FF13Rando/Randomizers/ShopRando.cs

```diff
@@ -22,6 +22,7 @@ public override void Load()
         }
         public override void Randomize(Action<int> progressSetter)
         {
+            //TODO: add option to lock in high quality upgrading components?
```

Comment:
items.csv has a "Force" trait to force items into the shop. ultra compacts and particle accelerators are there already.